### PR TITLE
feat: [PL-38926]: Fixed the copy entities script 

### PR DESCRIPTION
**Pull request by adiyaar24**
## Describe your changes

Added Readme file over here and changed order of execution.

## Checklist
- [ ] I've documented the changes in the PR description.
- [ ] I've tested this change either in PR or local environment.
- [ ] If hashcheck failed I followed [the checklist](https://harness.atlassian.net/wiki/spaces/DEL/pages/21016838831/PR+Codebasehash+Check+merge+checklist) and updated this PR description with my answers to make sure I'm not introducing any breaking changes.

## Comment Triggers
<details>
  <summary>Build triggers</summary>

- Feature build: `trigger feature-build`
- 
  Specific builds
  - `trigger manager`
  - `trigger dms`
  - `trigger ng_manager`
  - `trigger cvng `
  - `trigger cmdlib`
  - `trigger template_svc`
  - `trigger events_fmwrk_monitor`
  - `trigger event_server`
  - `trigger change_data_capture`
  -  Trigger multiple builds together: For eg: `trigger dms, manager`
- Immutable delegate `trigger publish-delegate`
</details>

## [Latest PR Check Triggers](https://github.com/harness/harness-core/blob/develop/.github/pull_request_template.md)

<details>
  <summary>PR Check triggers</summary>
You can run multiple PR check triggers by comma separating them in a single comment. e.g. `trigger ut0, ut1`

- Compile: `trigger compile`
- CodeformatCheckstyle: `trigger checkstylecodeformat`
  - CodeFormat: `trigger codeformat`
  - Checkstyle: `trigger checkstyle`
- MessageMetadata: `trigger messagecheck`
- File-Permission-Check: `trigger checkpermission`
- Recency: `trigger recency`
- BuildNumberMetadata: `trigger buildnum`
- Trigger CommonChecks: `trigger commonchecks`
- PMD: `trigger pmd`
- Copyright Check: `trigger copyrightcheck`
- Feature Name Check: `trigger featurenamecheck`
- UnitTests-ALL: `trigger uts`
- UnitTests-0: `trigger ut0`
- UnitTests-1: `trigger ut1`
- UnitTests-2: `trigger ut2`
- UnitTests-3: `trigger ut3`
- UnitTests-4: `trigger ut4`
- UnitTests-5: `trigger ut5`
- UnitTests-6: `trigger ut6`
- UnitTests-7: `trigger ut7`
- UnitTests-8: `trigger ut8`
- UnitTests-9: `trigger ut9`
- CodeBaseHash: `trigger codebasehash`
- CodeFormatCheckstyle: `trigger checkstylecodeformat`
- SonarScan: `trigger ss`
- GitLeaks: `trigger gitleaks`
- Trigger all Checks: `trigger smartchecks`
- Go Build: `trigger gobuild`
- Validate_Reviews: `trigger review`
- Module Dependency Check: `trigger mdc`
</details>

## PR check failures and solutions
https://harness.atlassian.net/wiki/spaces/BT/pages/21106884744/PR+Checks+-+Failures+and+Solutions

## [Contributor license agreement](https://github.com/harness/harness-core/blob/develop/CONTRIBUTOR_LICENSE_AGREEMENT.md)

- - -
This change is [Reviewable](https://reviewable.io/reviews/harness/harness-core/49509)

### CODE REVIEW

**Comment by adiyaar24** [comment]
trigger uts

File: scripts/copy-entities/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    - If the migration is being done in the same org and the SM are present at the org level then this is not needed
+
+
+## Limitations of the Script
+
+1. This script won’t migrate any inline secrets stored in any secret manager.
+2. All the remote Pipelines will be moved to Inline
+3. All the remote templates would be move to Inline
+4. If the SM is not migrated then the secrets present in that SM will also be not migrated
+   5.. Make sure that the identifier remains the same while creating the prerequisites data .
```
Comment (rijajoo) [inline scripts/copy-entities/README.md:16]: Let's fix this.

Comment (adiyaar24) [reply scripts/copy-entities/README.md]: Resolved

File: scripts/copy-entities/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  Cases when this is not needed
+    - If the SM are at account level then there is no need to do the same
+    - If the migration is being done in the same org and the SM are present at the org level then this is not needed
+
+
+## Limitations of the Script
+
+1. This script won’t migrate any inline secrets stored in any secret manager.
+2. All the remote Pipelines will be moved to Inline.
+3. All the remote templates would be move to Inline.
```
Comment (nataraja2018) [inline scripts/copy-entities/README.md:14]: moved to in line no 14

**Review by nataraja2018 — COMMENTED** [review]
Look into the comment

**Review by nataraja2018 — APPROVED** [review]
:lgtm:

> *[Reviewable](https://reviewable.io/reviews/harness/harness-core/49509#-:-NYRusYd3hEMNSsJkNkE:bnfp4nl)* status: 2 file(s) to review. 1. unresolved discussions. 1 of 2 approvals obtained.

**Pull request merged**